### DISCUSSION TITLE
How to disable/hide header information

### DISCUSSION THREAD

**Discussion by balait4** [Kong/kong#11520] [Help] [answered]
Hi,

Using kong 3.3.0 standalone installation (DB-less) and like to know how can I hide/disable kong header in the request. 
Right now I'm getting, how can I hide `server: nginx/1.16.1` and `via: kong/3.3.0` in this request. 
I tried setting the env `KONG_HEADER: OFF` but didn't help. Any workaround with this kong version?
```
HTTP/2 200
content-type: text/html; charset=UTF-8
content-length: 558017
vary: Accept-Encoding
server: nginx/1.16.1
date: Thu, 07 Sep 2023 08:36:51 GMT
last-modified: Fri, 28 Jul 2023 10:44:32 GMT
etag: "64c39c10-883c1"
accept-ranges: bytes
x-kong-upstream-latency: 1
x-kong-proxy-latency: 0
via: kong/3.3.0
strict-transport-security: max-age=31536000
```

**Comment by shettyh** [comment, marked as answer]
Try setting [this](https://docs.konghq.com/gateway/latest/reference/configuration/#headers) as off or set env `KONG_HEADERS=off`
